pkgs/servers/invidious/update.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl crystal2nix jq git gnused nix nix-prefetch-git nix-update
+invidious_git='/var/tmp/invidious.git'
+
+set -euo pipefail
+
```

Comment:
I think this should explicitly change the directory to the root of the package, because otherwise `nix-shell maintainers/scripts/update.nix --argstr package invidious` fails.

```suggestion
cd "$(dirname "${BASH_SOURCE[0]}")"

```